packages/react-integration/demo-app-ts/tsconfig.json

```diff
@@ -18,6 +18,9 @@
     "noEmit": true,
     "jsx": "preserve",
     "strict": true,
+    "strictPropertyInitialization": false,
+    "strictNullChecks": false,
```

Comment:
There should be followup work so the topology demo can work in a stricter setting